src/lib/glee.ts

```diff
@@ -37,8 +37,10 @@ export default class Glee extends EventEmitter {
    *
    * @param {Object} [options={}]
    */
-  constructor (options:GleeConfig = {}) {
-    super()
+  constructor(options: GleeConfig = {}) {
+    super({ captureRejections: true })
+
+    this.on('error', (error) => console.log(error))
```

Comment:
@KhudaDad414 let me know if this will do, for now, just logging the error right now.